frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -37,7 +37,7 @@ export const filterSCWithoutNoProv = (sc: StorageClassResourceKind) =>
 export const getTotalDeviceCapacity = (list: Discoveries[]) => {
   const totalCapacity = list.reduce((res, device) => {
     if (device?.status?.state === AVAILABLE) {
-      const capacity = Number(convertToBaseValue(device.size));
+      const capacity = Number(convertToBaseValue(device.size.toString()));
```

Comment:
Size is already coming as `integer` value only so we can directly use it.
 `convertToBaseValue` will not be required.
```suggestion
      const capacity = device.size;
```